fmriprep/cli/run.py

```diff
@@ -538,6 +538,13 @@ def build_workflow(opts, retval):
         retval['return_code'] = 1
         return retval
 
+    if bids_dir in work_dir.parents:
+        build_log.error(
+            'The selected working directory is a subdirectory of the input BIDS folder. '
+            'Please modify the output path.')
+        retval['return_code'] = 1
+        return retval
```

Comment:
Should probably do a similar thing in `fmriprep-docker`, since we could do:

```
fmriprep-docker /a /b participant -w /a/work
```

And it would translate to:

```Shell
docker run --rm -it -v /a:/data:ro -v /b:/out -v /a/work:/scratch \
    poldracklab/fmriprep:<version> /data /out participant -w /scratch
```

And not hit this condition.

Reply:
Nice catch.